app/src/routes/$id.inntekt-og-refusjon.tsx

```diff
@@ -8,80 +8,122 @@ import {
   ReadMore,
   VStack,
 } from "@navikt/ds-react";
-import { createFileRoute, Link } from "@tanstack/react-router";
-import { format } from "date-fns";
-import { nb } from "date-fns/locale";
+import { Link, createFileRoute, useNavigate } from "@tanstack/react-router";
+import { FormProvider, useForm, useFormContext } from "react-hook-form";
 
 import type { OpplysningerDto } from "~/api/queries.ts";
 import { hentOpplysningerData } from "~/api/queries.ts";
+import { InformasjonsseksjonMedKilde } from "~/features/InformasjonsseksjonMedKilde";
+import { useInntektsmeldingSkjema } from "~/features/InntektsmeldingSkjemaState";
 import { Fremgangsindikator } from "~/features/skjema-moduler/Fremgangsindikator.tsx";
 import { Inntekt } from "~/features/skjema-moduler/Inntekt.tsx";
-import { InformasjonsseksjonMedKilde } from "~/features/skjema-moduler/PersonOgSelskapsInformasjonSeksjon.tsx";
-import { capitalizeSetning, leggTilGenitiv } from "~/utils.ts";
+
+import {
+  capitalizeSetning,
+  formatDatoLang,
+  formatYtelsesnavn,
+  leggTilGenitiv,
+} from "~/utils.ts";
 
 export const Route = createFileRoute("/$id/inntekt-og-refusjon")({
   component: InntektOgRefusjon,
   loader: ({ params }) => hentOpplysningerData(params.id),
 });
 
+type InntektOgRefusjonForm = {
+  skalRefunderes: "ja" | "nei";
+  misterNaturalytelser: "ja" | "nei";
+};
+
 function InntektOgRefusjon() {
   const opplysninger = Route.useLoaderData();
 
+  const { inntektsmeldingSkjemaState, setInntektsmeldingSkjemaState } =
+    useInntektsmeldingSkjema();
+  const formMethods = useForm<InntektOgRefusjonForm>({
+    defaultValues: {
+      skalRefunderes:
+        inntektsmeldingSkjemaState.skalRefunderes === undefined
+          ? undefined
+          : inntektsmeldingSkjemaState.skalRefunderes
+            ? "ja"
+            : "nei",
```

Comment:
Man kan jo vurdere om man skal lage en egen util for å løse denne generelt, men det er vel bare disse to hvor det vil være et issue.

Reply:
På min branch satte jeg value til true/false heller enn "ja", "nei". Jeg minnes at det ble noe funky når man hadde value=true på radio, men ser ut til at det spiller helt fint

```
<RadioGroupWrapped
        legend="Har den ansatte naturalytelser som faller bort ved fraværet?"
        name="misterNaturalytelser"
        rules={{
          validate: (value: boolean | null) =>
            value === null ? "Må oppgis" : true,
        }}
      >
        <Radio value={true}>Ja</Radio>
        <Radio value={false}>Nei</Radio>
      </RadioGroupWrapped>
```

Reply:
Ah okei da blir det jo enklere. Fiks det når du merger inn din branch du!